docs/advanced/prometheus.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Monitoring KubeEdge Edge Nodes with Prometheus
+sidebar_position: 6
+---
+# Monitoring KubeEdge Edge Nodes with Prometheus
+
+## Environment Information
+
+| Component  | Version                            |
+| ---------- | ---------------------------------- |
+| containerd | 1.7.2                              |
+| k8s        | 1.26.0                             |
+| KubeEdge   | 1.15.1 or 1.17.0                   |
+| Jetson型号 | NVIDIA Jetson Xavier NX (16GB ram) |
+
+## Deploying Prometheus
+
+We can quickly install using the [Helm Charts](https://prometheus-community.github.io/helm-charts/)  of [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus), or we can install it manually.
+
+It is important to pay attention to the compatibility between the Kubernetes version and kube-prometheus.
+
+```shell
+git clone https://github.com/prometheus-operator/kube-prometheus.git
+cd kube-prometheus
+kubectl apply --server-side -f manifests/setup
+kubectl wait \
+	--for condition=Established \
+	--all CustomResourceDefinition \
+	--namespace=monitoring
+kubectl apply -f manifests/
+```
+
+You can see that a ClusterIP type Service has been created for grafana, alertmanager, and prometheus. Of course, if we want to access these two services from the Internet, we can create the corresponding Ingress objects or use NodePort type Services. Here, for simplicity, we directly use NodePort type services. Edit the 3 Services of grafana, alertmanager-main, and prometheus-k8s to change the service type to NodePort:
+
+![image-20240524161614721](https://zhuyaguang-1308110266.cos.ap-shanghai.myqcloud.com/img/image-20240524161614721.png)
```

Comment:
images could put at https://github.com/kubeedge/website/tree/master/static/img, you can add a new directory named `advanced`